files/build/versions/build/build-sonic-slave-stretch/versions-deb-stretch

```diff
@@ -0,0 +1,23 @@
+innovium-sai==0.11.1
+innovium-sai-headers==0.11.1
+libhiredis-dev==0.14.0-3~bpo9+1
+libhiredis0.14==0.14.0-3~bpo9+1
+libnl-3-200==3.5.0-1
+libnl-3-dev==3.5.0-1
+libnl-cli-3-200==3.5.0-1
+libnl-cli-3-dev==3.5.0-1
+libnl-genl-3-200==3.5.0-1
+libnl-genl-3-dev==3.5.0-1
+libnl-nf-3-200==3.5.0-1
+libnl-nf-3-dev==3.5.0-1
+libnl-route-3-200==3.5.0-1
+libnl-route-3-dev==3.5.0-1
+libsainps==1.5.0
+libsairedis==1.0.0
+libsaithrift-dev==0.9.4
+libswsscommon==1.0.0
+libswsscommon-dev==1.0.0
```

Comment:
Many packages are build targets inside slave containers. By design, should we save their versions? If not, can you fix the code which generated them?

Reply:
Hi Qi, it is not necessary to save the versions, but it is no harm, we do not need to additional actions to filter them out.

Reply:
could you add a github issue? I know it is no harm right now.

Reply:
I have created an issue https://github.com/Azure/sonic-buildimage/issues/6851